content/participate/report-issue.adoc

```diff
@@ -98,13 +118,24 @@ for descriptions of the levels. *This is not how urgently* *_you_* *want
 it fixed*. Try to be realistic: Most bugs should be Minor or Major
 priority. If you found a workaround, reduce the priority, even if it's a
 severe problem otherwise.
-* As *Component*, select _core_ if it's a problem with Jenkins itself
-and not any of the plugins. Otherwise, select the name of the plugin;
-all of them should have a related component. You can also select
-multiple components e.g. if the problem is in the interaction of the
-components, or when you're not sure.
 * Leave the *Assignee* field to automatic.
 
+=== Reporting an issue on GitHub 
+You need to have a Github account. 
+link:https://docs.github.com/en/get-started/onboarding/getting-started-with-your-github-account[Register an account] if you haven't already.
+
+* If you are redirected to the Github issue tracker from the link:https://plugins.jenkins.io/[Plugin Site], you will see something like this:
++
+[.boxshadow]
+image:/images/participate/github-issues.png[alt="Github Issue Page",width=80%]
+
+* Select *Get started* next to the type of issue you'd like to open.
++
+[.boxshadow]
+image:/images/participate/get-started.png[alt="Get Started Button",width=40%,align="center"]
+
+* After that, fill in the details of the issue according to the Github issue template provided for each type of issue.
+* Don't forget to subscribe to the issue so you'll get notifications regarding any updates on the issue.
 [[Howtoreportanissue-WhatinformationtoprovideforEnvironmentandDescription]]
 === What information to provide for Environment and Description
```

Comment:
```suggestion
[[Howtoreportanissue-WhatinformationtoprovideforEnvironmentandDescription]]

=== What information to provide for Environment and Description
```
there needs to be an empty line between the anchor and the header so that the formatting can properly render.